packages/test/harness/src/host/common/host/sendTab.js

```diff
@@ -1,7 +1,17 @@
 const { Key } = require('selenium-webdriver');
 
 module.exports = webDriver => {
-  return function sendTab() {
-    return webDriver.actions().sendKeys(Key.TAB).perform();
+  return function sendTab(count = 1) {
+    if (!(count >= 1 && count <= 100)) {
+      throw new Error('First argument passed to sendTab() must between 1 and 100.');
+    }
+
+    let actions = webDriver.actions();
+
+    while (count--) {
+      actions = actions.sendKeys(Key.TAB);
```

Comment:
This doesn't need an exit param?

Reply:
Exit parameter?

- `count` must be 1 to 100
- If `count` is `0` *before* decrement, then stop